### DISCUSSION TITLE
Connectbox - Enhanced Interface NEO

### DISCUSSION THREAD

**Discussion by kirkdwilson** [ConnectBox/connectbox-pi#335] [General]
This release is a beta release of the NEO platform code of the Connectbox with a new enhanced interface.  It supports multi-lingual content by using folders with folder titles using the ISO codes that match browser header codes from [RFC 5646] 
[https://www.rfc-editor.org/rfc/rfc5646] and controlled by IANA.  This is typically a two character code such as "en" for English, "es" for Spanish, etc.  The enhanced interface will isolate these folders based on language.  There is a language selector at the top right of the page.  Additionally, thumbnails are created for all media content which are stored on the media.  Video thumbnails are created for any media that the browser can play.  Typically this is H269 encoded video mp4 with aac audio.  PDF reading is supported as well for viewing or downloading.  Please provide feedback on the interface and we see this as a major enhancement for the device.  

Additionally, a client connection can be made to an access point for internet access.  This allows the device to  phone home to a central console to get updated content as well as updated  parameters and upload statistical information..  We will be releasing this code to spin up a server to manage  multiple connect boxes shortly.

Note that on initial boot of the new code there will be at least three reboots before the device comes up.  So please give it plenty of time.
To create the thumbnail files and index of the content may take a while as well so the initial install of new content will take a while before it comes up on the interface.  However, the index and thumbnails are stored on the media so if it has previously been indexed and thumbnailed it will load quickly.

<hr /><em>This discussion was created from the release <a href='https://github.com/ConnectBox/connectbox-pi/releases/tag/'>Connectbox - Enhanced Interface NEO</a>.</em>

**Comment by montieroland** [comment]
Is there an install and configuration doc available?

**Reply by montieroland** [reply]
How do we know when the installation is complete?  Does red light for the micro sd card ever go out?

**Comment by kirkdwilson** [comment]
No the red light may or may not go out.  Depends on the state of the system.

 

Its install is complete after about 3 reboots.  You will see the green disk led light effectively  go out for a long period of time with just a small ficker here and there.

 

If you have a monitor connected its easier to see the end.

 

Kirk

**Comment by kirkdwilson** [comment]
I have notyet written a new document for the new release code.  We have the old document but there are many changes since we changed interfaces.

 

So I’m sorry not yet.  It will be arriving in the next weeks.

 

Kirk

**Comment by montieroland** [comment]
No problem, thanks for the heads ul

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by montieroland** [comment]
Morning,

Do you have the doc for how to install wordpress on the latest version?

Which version should I down load. I have the wifi connect box from you.

Montie

Montie Roland
***@***.***
Sanford, NC

A candle loses nothing by lighting another candle

Blog and Podcast - www.montie.com
Montie Gear - www.montiegear.com
Simply Us Farm - www.simplyusfarm.com

------- Original Message -------

**Comment by kirkdwilson** [comment]
Dear Montie;

You’ll find the releases at: https://github.com/ConnectBox/connectbox-pi/releases

Since your putting wordpress on it I would use  you could simply use the NEOPi Wordpress Release -Armbian Focal of Nov 10, 2021 which has wordpress already installed.  

 

If you want to use a solid NEO release then the NEO-103121 release.  Then install wordpress yourself.

 

Regards;

 

Kirk

**Comment by montieroland** [comment]
Do you know when you will have an updated version that includes Wordpress?

Montie

Montie Roland
***@***.***
Sanford, NC

A candle loses nothing by lighting another candle

Blog and Podcast - www.montie.com
Montie Gear - www.montiegear.com
Simply Us Farm - www.simplyusfarm.com

------- Original Message -------

**Comment by kirkdwilson** [comment]
Montie;

It may be a couple of weeks.  You can do the installation yourself if you like following the guide.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
